### DISCUSSION TITLE
Monitoring the node values on the server side

### DISCUSSION THREAD

**Discussion by cabloke** [eclipse-milo/milo#1004] [General]
Hello Everyone,

I successfully subscribed to nodes using the SubscriptionExample on client side，now I would like to monitor the node values on server side,how can I achieve it through API ? 

Thanks,
cabloke

**Comment by kevinherron** [comment]
There are probably a couple ways you could achieve this on the server-side... but what exactly are you trying to do? 

You can add an `AttributeObserver` to a `UaNode`, but usually in the server implementation you are aware of when a value changes.

**Reply by cabloke** [reply]
I want the server - side to monitor the client's write of node values，similar to the client's subscription.
Thanks for your suggestions!
